src/components/v2/V2Create/tabs/ReviewDeployTab/FundingAttributes.tsx

```diff
@@ -159,6 +169,32 @@ export function IssuanceRateStatistic({
   )
 }
 
+export function InflationRateStatistic({
+  inflationRate,
+  isInitial,
+}: {
+  inflationRate: string
+  isInitial?: boolean
+}) {
+  return (
+    <Statistic
+      title={
+        <TooltipLabel
+          label={isInitial ? t`Initial inflation rate` : t`Inflation rate`}
+          tip={
+            <Trans>
+              <strong>Total project tokens minted</strong> as a result of 1 ETH
+              being contributed. This can change over time according to the
+              discount rate and reserved tokens amount of future funding cycles.
```

Comment:
```suggestion
              <strong>Total project tokens minted</strong> when 1 ETH
              is contributed. This can change over time according to the
              discount rate and reserved tokens amount of future funding cycles.
```